packages/anoncreds/src/protocols/proofs/v1/V1ProofProtocol.ts

```diff
@@ -220,7 +222,7 @@ export class V1ProofProtocol extends BaseProofProtocol implements ProofProtocol<
       await didCommMessageRepository.saveOrUpdateAgentMessage(agentContext, {
         agentMessage: proposalMessage,
         associatedRecordId: proofRecord.id,
-        role: DidCommMessageRole.Sender,
+        role: DidCommMessageRole.Receiver,
```

Comment:
I think this one was set incorrectly, a double-check at the review would be good!